testdata/ppc-expected-profiles/profile2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "mg": "must-gather.sno",
```

Comment:
For more clarity here, I would recommend using the flag names here ( as we use when we run PPC) i.e. `must-gather-dir-path` instead of `mg`, `mcp-name` instead of `mcp`,  `reserved-cpu-count` instead of `reserved` and `rt-kernel` instead of  `rt`.

Reply:
Sure, I can do that

Reply:
done